x-pack/legacy/plugins/ml/public/application/jobs/new_job/pages/new_job/page.tsx

```diff
@@ -86,6 +86,10 @@ export const Page: FC<PageProps> = ({ existingJobsAndGroups, jobType }) => {
       // auto set the time range based on the index
       autoSetTimeRange = isAdvancedJobCreator(jobCreator);
     }
+
+    if (mlJobService.tempJobCloningObjects.calendars) {
+      jobCreator.calendars = mlJobService.tempJobCloningObjects.calendars;
```

Comment:
`mlJobService.tempJobCloningObjects.calendars` should be set to `undefined` after being read

Reply:
/giphy magic

Reply:
https://github.com/elastic/kibana/pull/51281/commits/b153888e4c30490dfcbf2b7f0299eb54daea65d9